site/docs/guides/admin/configuring-the-omag-server-platform.md

```diff
@@ -261,6 +261,17 @@ When the platform shuts down, if any of the servers that were in the startup lis
 
     This is the default setting.
 
+## Additional HTTP headers
+
+By default, the security connectors use only basic authentication. To improve the security checks there is the possibility to add additional headers that will be passed to the security plugins and even further, propagated on federated queries.
+
+How it works:
+
+When `header.name.list` is set, the HttpRequestHeadersFilter puts all specified headers in a ThreadLocal variable. By default, this list is empty.
```

Comment:
This 'how it works' section is mostly relevant to the security connector developer -- so I think this detail belongs there. It is not related to the configuration of the platform. That stops with specifying the list -- and of course defining the user of a security connector

Reply:
@planetf1 I agreeI see [https://egeria-project.org/guides/developer/runtime-connectors/server-metadata-security-connector/](https://egeria-project.org/guides/developer/runtime-connectors/server-metadata-security-connector/). There is a link to Writing a security connector at the bottom of the page - which points to itself. I suggest adding a new 'Writing a Security connector' page (which seems to be intended) and add this information.

Reply:
I agree + we are already in process of writing new sample security connector that should provide "how-to" for this. we will remove the "How it works section"

Reply:
actually, how it works should be in the javadocs of the component implementing this (the filter).
On the security connector side we are documenting the usage aspect with JWT  (only one use case for this platform feature)